_devExtensions/updateBefore-controller.js

```diff
@@ -0,0 +1,46 @@
+const bcrypt = require('bcryptjs')
+const BS3Database = require('better-sqlite3')
+const path = require('path')
+
+const getUserById = (db, userId) => {
+  const query = `
+    SELECT createdAt, createdBy, departedDate, id, is_superuser, lastUpdatedAt, lockoutAttempts, name, role, updateBefore, username
+    FROM _users
+    WHERE id = ?;
+  `
+  return db.prepare(query).get(userId)
+}
+
```

Comment:
@TahaKhanAbdalli - the following string clears the `updateBefore` field. Should it be called `clearUserUpdateBefore`?

Reply:
Sure